src/main/java/org/sagebionetworks/web/client/widget/entity/tabs/FilesTab.java

```diff
@@ -322,12 +324,12 @@ public void setTargetBundle(EntityBundle bundle) {
 		view.setMetadataVisible(isMetadataVisible);
 		if (isMetadataVisible) {
 			initActionMenu(bundle);
```

Comment:
The first step in the pipeline to making sure the EntityActionController shows the "Can't edit metadata message" ie how it gets version information